apps/src/gamelab/GameLabVisualizationColumn.jsx

```diff
@@ -37,6 +41,19 @@ var GameLabVisualizationColumn = React.createClass({
     };
   },
 
+  componentWillReceiveProps(nextProps) {
+    // Use jQuery to turn on and off the grid since it lives in a protected div
+    if (nextProps.showGrid !== this.props.showGrid) {
+      if (nextProps.showGrid) {
+        $("#grid-checkbox")[0].className = "fa fa-check-square-o";
+        $("#grid-overlay")[0].style.display = '';
+      } else {
+        $("#grid-checkbox")[0].className = "fa fa-square-o";
+        $("#grid-overlay")[0].style.display = 'none';
+      }
+    }
+  },
```

Comment:
GridOverlay is also in a protected div, so it won't ever receive new props, which is why is has to be modified here. Since they both need to be modified in this component, I just decided to keep them consistent by updating the classname in the same way.